libs/watt/package/picker/datepicker/watt-datepicker.component.ts

```diff
@@ -96,14 +98,15 @@ export class WattDatepickerComponent extends WattPickerBase implements Validator
   protected override changeDetectionRef = inject(ChangeDetectorRef);
   protected override ngControl = inject(NgControl, { optional: true, self: true });
   private localeService = inject(WattLocaleService);
-  private locale: WattSupportedLocales = inject(LOCALE_ID) as WattSupportedLocales;
+  private locale = inject<WattSupportedLocales>(LOCALE_ID);
 
   max = input<Date>();
   min = input<Date>();
   rangeMonthOnlyMode = input(false);
-
-  @Input() startAt: Date | null = null;
-  @Input() label = '';
+  startAt = input<Date | null>(null);
+  label = input<string>('');
+  dateClass = input<MatCalendarCellClassFunction<Date>>(() => '');
+  canStepThroughDays = input(false, { transform: booleanAttribute });
```

Comment:
Question: Why is booleanAttribute needed here?

Reply:
The "traditional" `[canStepThroughDays]="true"` is still possible.